LanguageSentimentText.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * DO NOT EDIT! This is a generated sample ("Request",  "language_sentiment_text")
+ */
+
+// sample-metadata
+//   title: Analyzing Sentiment
+//   description: Analyzing Sentiment in a String
+//   usage: php samples/V1/LanguageSentimentText.php [--text_content "I am so happy and joyful."]
+// [START language_sentiment_text]
+require __DIR__.'/../../vendor/autoload.php';
```

Comment:
Should the require statement be shown in the snippet?

I see that it is generally _not shown_, only the `use` statements

Reply:
I'd lean no, because the path is context dependent. If viewing this sample from the cloud site this path doesn't mean much. It might be nice to say instead have a way of saying "make sure you are requiring your the composer autoload file".